### DISCUSSION TITLE
Magisk加载了一个错误的模块导致WSA不能启动

### DISCUSSION THREAD

**Discussion by sunshinewithmoonlight** [LSPosed/MagiskOnWSALocal#334] [Q&A] [answered]
Magisk加载了一个错误的模块导致WSA不能正常启动，咋整啊，里面的数据不想丢啊......

**Comment by Howard20181** [comment, marked as answer]
没办法不开机解密data 

所以唯一可能有救的前提是之前授权过adb shell的root权限，然后在bootloop那时候手快进去禁用掉模块 

除此之外还可以编辑init.rc在里面禁用模块，这个也算方便，因为构建脚本就有这个步骤，你在post-fs-data时执行就可以
